cmake/Modules/FindIrrlicht.cmake

```diff
@@ -3,31 +3,25 @@ mark_as_advanced(IRRLICHT_DLL)
 
 # Find include directory and libraries
 
-# find our fork first, then upstream (TODO: remove this?)
-foreach(libname IN ITEMS IrrlichtMt Irrlicht)
-	string(TOLOWER "${libname}" libname2)
-
-	find_path(IRRLICHT_INCLUDE_DIR NAMES irrlicht.h
-		DOC "Path to the directory with IrrlichtMt includes"
-		PATHS
-		/usr/local/include/${libname2}
-		/usr/include/${libname2}
-		/system/develop/headers/${libname2} #Haiku
-		PATH_SUFFIXES "include/${libname2}"
-	)
-
-	find_library(IRRLICHT_LIBRARY NAMES lib${libname} ${libname}
-		DOC "Path to the IrrlichtMt library file"
-		PATHS
-		/usr/local/lib
-		/usr/lib
-		/system/develop/lib # Haiku
-	)
-
-	if(IRRLICHT_INCLUDE_DIR OR IRRLICHT_LIBRARY)
-		break()
-	endif()
-endforeach()
+set(libname Irrlicht)
+string(TOLOWER "${libname}" libname2)
+
+find_path(IRRLICHT_INCLUDE_DIR NAMES irrlicht.h
+	DOC "Path to the directory with IrrlichtMt includes"
+	PATHS
+	/usr/local/include/${libname2}
+	/usr/include/${libname2}
+	/system/develop/headers/${libname2} #Haiku
+	PATH_SUFFIXES "include/${libname2}"
+)
```

Comment:
I'd inline the `find_path(IRRLICHT_INCLUDE_DIR ...)` and first "handholding" check, the rest of this file isn't of any use anymore.

Reply:
I've put this in a module and included it, to keep things organized.

Reply:
Remove `FindIrrlicht.cmake` then?